### PR TITLE
Scorm analytics synchronize issue

### PR DESCRIPTION
**Pull request by Cory-Christensen**
When hitting the back button on analytics side, any new scorm packages will be deleted.
This prevents the app from synchronizing when hitting the analytics back button, by using the hashHistory query prop. This solution leaves a bad taste in my mouth so if you can think of a better way let's talk :)

### CODE REVIEW

File: client/js/scorm/components/scorm/_scorm_index.jsx [hunk 1]
```diff
@@ -59,6 +64,10 @@ export class ScormIndex extends React.Component {
   }
 
   synchronize() {
+    if (this.props.location && this.props.location.query.noSync) {
+      this.setState({ synced: true });
+      return;
```
Comment (bfcoder) [inline client/js/scorm/components/scorm/_scorm_index.jsx]: Consider putting the return on the previous line?

Comment (Cory-Christensen) [inline client/js/scorm/components/scorm/_scorm_index.jsx]: The component shows ```loading...``` until the synced state is set to true.

Comment (bfcoder) [inline client/js/scorm/components/scorm/_scorm_index.jsx]: But it won't work if you do
```javascript
return this.setState({ synced: true });
```
?
But maybe that gives the wrong impression of this method as it isn't suppose to return a value.

Would it make sense to put the `forEach` into an else? Or maybe into an `if !` sort of thing? As it is, you are duplicating setting the state to `synced: true`

Comment (Cory-Christensen) [inline client/js/scorm/components/scorm/_scorm_index.jsx]: It doesn't duplicate - it returns from the synchronize function. We could consider putting the forEach into an else, but it shouldn't make a difference one way or another.

Comment (bfcoder) [inline client/js/scorm/components/scorm/_scorm_index.jsx]: I mean code duplication, not getting set twice.

Comment (Cory-Christensen) [inline client/js/scorm/components/scorm/_scorm_index.jsx]: Ah, I get it. Sure, I think I've got a way

**Pull request merged**